lib/data/guides.json

```
@@ -1,4 +1,14 @@
 [
+  {
+    "title": "The Ultimate Guide to Next.js and Vercel Logging with Logflare",
```

I think we avoid using catchy words like "Ultimate" in our other guides, so can we change it to something like this:

```suggestion
    "title": "Next.js and Vercel Logging with Logflare",
```